src/Native/Runtime/i386/AllocFast.S

```diff
@@ -0,0 +1 @@
+// TODO: Implement
```

Comment:
A nit - could you please add the standard licence header to all of the new .S files?

Reply:
This is addressed, git diff is not showing here, full diff does: https://github.com/dotnet/corert/pull/4637/files#diff-240395baa0c54b3dd111b2a1f3645e4d